src/App.jsx

```diff
@@ -14,57 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const [message, setMessage] = useState('Jam is selected');
```

Comment:
Consider initializing the `good` and `message` states with empty values or default values that indicate no selection. This will make the initial state more consistent with the behavior when no goods are selected.